src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -1034,7 +1034,21 @@ private static void updateVelocity(Entity e, double scale)
             updatePosition(e, e.getX(), e.getY(), e.getZ(), e.yaw, MathHelper.clamp((float)NumericValue.asNumber(v).getDouble(), -90, 90));
         });
 
-        //"look"
+        put("look", (e, v) -> {
+            if (!(v instanceof ListValue)) throw new InternalExpressionException("Expected a list of 3 parameters as a second argument");
+            List<Value> vec = ((ListValue)v).getItems();
+            float x = NumericValue.asNumber(vec.get(0)).getFloat();
+            float y = NumericValue.asNumber(vec.get(1)).getFloat();
+            float z = NumericValue.asNumber(vec.get(2)).getFloat();
+            float l = MathHelper.sqrt(x*x + y*y + z*z);
```

Comment:
I would protect from zero division here. Also yaw is undef afaik if you look straight up or down - in that case just make sure its ... zero?

Reply:
Yeah  i was wondering what a vector that cant properly define pitch or yaw should do. Same case for looking straight down or up, then the yaw would always have the same value. I was thinking about checking for those cases and keeping player's yaw, or just set yaw to the same value, as it is now.

And yeah docs, completely forgot that